### DISCUSSION TITLE
Google Earth

### DISCUSSION THREAD

**Discussion by karellinseele** [SafeExamBrowser/seb-win-refactoring#558] [General]
Using Google Maps in SEB works as a charm. However since januari **Google Earth** doesn't seem to work anymore in SEB. The page keeps loading. I'm using version 3.4.1.

<img width="960" alt="Google Earth" src="https://user-images.githubusercontent.com/123454796/214257240-d2be2100-a644-4943-8147-0c0e0d6f9c46.png">

Any ideas what might resolve this? 

Thanks!

**Comment by dbuechel** [comment]
Could you elaborate how exactly you're trying to use Google Earth (step-by-step instruction)? And did you verify that Google Earth works correctly outside of SEB?

**Reply by karellinseele** [reply]
Hi Damian

Thank you for your reply. 

I verified that Google Earth is working fine outside of SEB. 

The start-url is a link to a[ Sway-page](https://sway.office.com/JHJIToUzMnICOeaH?ref=Link&loc=mysways) I made. This Sway contains two links my students need during an exam. The second link is Google Earth. 

I whitelisted the link to Google Earth in the tab Network.
<img width="553" alt="image" src="https://user-images.githubusercontent.com/123454796/217185868-587075b1-ede0-4e0b-9d01-2db5dbaf75e9.png">

This is a link to my SEB-file. The passwords are start, stop & admin.
[https://olvp-my.sharepoint.com/:u:/g/personal/karel_linseele_vlot_be/ER4gsSqpHFlIh3cW6iarzOsB0nxl2lDZpoTZvxry78pntw?e=S6KY3f](url)

thanks you for your help in solving my issue

**Reply by dbuechel** [reply]
I guess the underlying reason is related to the following log entry in the browser log:

```[0207/150811.075:INFO:CONSOLE(1)] "Uncaught ReferenceError: SharedArrayBuffer is not defined", source: https://earth.google.com/static/9.181.0.1/earthwasm.js (1)```

It appears that `SharedArrayBuffer` has been disabled due to security concerns, starting with Chromium 92 (see e.g. https://stackoverflow.com/questions/67914209/uncaught-referenceerror-sharedarraybuffer-is-not-defined-since-chrome-92). Hence, we cannot enable it by default. But we'll be adding the possibility to specify custom browser command line arguments, once we've done that, you could manually enable `SharedArrayBuffer` as explained in the SO thread above.

**Reply by karellinseele** [reply]
Thank you Damian!
